src/Servers/HttpSys/src/MessagePump.cs

```diff
@@ -134,7 +134,8 @@ public Task StartAsync<TContext>(IHttpApplication<TContext> application, Cancell
                 _serverAddresses.Addresses.Add(prefix.FullPrefix);
             }
 
-            ActivateRequestProcessingLimits();
+            // Dispatch to get off the SynchronizationContext and use UnsafeQueueUserWorkItem to avoid capturing the ExecutionContext
+            ThreadPool.UnsafeQueueUserWorkItem(state => state.ActivateRequestProcessingLimits(), this, preferLocal: false);
```

Comment:
Possible to drop the async void here? https://github.com/dotnet/aspnetcore/blob/b795ac3546eb3e2f47a01a64feb3020794ca33bb/src/Servers/HttpSys/src/MessagePump.cs#L177

Reply:
Yes async void is 😄

Reply:
thx